docs/xenserver-automated-certification-kit-guide.md

```diff
@@ -639,27 +639,27 @@ Examples:
 1.Run only Crash Dump test class
 
 ```
-   # python ack_cli.py -n network.conf –o \
+   # ./ack_cli.py -n network.conf –o \
    "run_classes=operations_tests.CrashDumpTestClass"  

```

Comment:
Check "–o"

Reply:
I have updated line 642 and line 648 to short "-".